app/assets/stylesheets/components/_highlight-boxes.scss

```diff
@@ -11,7 +11,7 @@
   width: 33%;
   min-height: 150px;
```

Comment:
Again, I may not be understanding this fully, but playing around with the examples this line doesn't seem to make any difference. I've just noticed the parent is using flexbox, which would explain it - I think by default all elements in a flex element have the same height i.e. the height of the tallest element. So this line is redundant, I believe.